utils/vlog/vlog_to_pbtype.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python3
+
+"""\
+Convert a Verilog simulation model to a VPR `pb_type.xml`
+
+The following are allowed on a top level module:
+
+    - `(* TYPE="bel|blackbox" *)` : specify the type of the module
```

Comment:
Actually, now I think about it -- we are confusing things here; there are two types of objects;
 * `BEL`s - Things which can't be decomposed.
 * `BLOCK`s - An object which contains multiple things (IE more blocks or other `BEL`s).

A `bel` is either;
 * a blackbox (`BEL_BB`)
 * one of the primitive types (`BEL_LT` / `BEL_RX` / `BEL_MX` / `BEL_MM` / `BEL_FF`).

See this doc -> https://github.com/mithro/symbiflow-arch-defs/tree/working-on-fasm#names

We should be able to use the `(* blackbox *)` property you used above and any module which is a `(* backbox *)` and doesn't have any `(* ALTERNATIVE_TO=xxxx *)` modes is thus a "BEL_BB" module.